surveys/survey-validator/lib/survey_validator.dart

```diff
@@ -35,6 +85,7 @@ void main() {
     final description = surveyObject['description'] as String;
     final snoozeForMinutes = surveyObject['snoozeForMinutes'] as int;
     final samplingRate = surveyObject['samplingRate'] as double;
+    final excludeDashToolsList = surveyObject['excludeDashTools'] as List;
```

Comment:
Oh. One extra thing to consider is that this will fail if the exclude list is missing. Not sure if that's intentional, but is there a case where it won't be needed and could just be dropped?